src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -14,6 +14,7 @@
             <argument type="collection" />
             <argument type="collection" />
         </service>
+        <service id="Symfony\Component\Serializer\SerializerInterface" alias="serializer" public="false" />
```

Comment:
I would also mark `Symfony\Component\Serializer\NormalizerInterface`, `Symfony\Component\Serializer\DenormalizerInterface`, `Symfony\Component\Serializer\EncoderInterface` and `Symfony\Component\Serializer\DecoderInterface` as aliases of the serializer.